Makefile

```diff
@@ -189,18 +201,27 @@ $(BUILD_DIRECTORY)/node-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies/node_modu
 		-t node \
 		-s "$(TARGET_PLATFORM)"
 
-$(BUILD_DIRECTORY)/electron-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies/bower_components: bower.json \
-	| $(BUILD_DIRECTORY)/electron-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies
-	./scripts/build/dependencies-bower.sh -p -x $|
-
 $(BUILD_DIRECTORY)/electron-$(TARGET_PLATFORM)-$(APPLICATION_VERSION)-$(TARGET_ARCH)-app: \
 	$(BUILD_DIRECTORY)/electron-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies/node_modules \
-	$(BUILD_DIRECTORY)/electron-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies/bower_components \
-	| $(BUILD_DIRECTORY)
+	| $(BUILD_DIRECTORY) $(BUILD_TEMPORARY_DIRECTORY)
 	./scripts/build/electron-create-resources-app.sh -s . -o $@ \
 		-v $(APPLICATION_VERSION) \
 		-f "$(APPLICATION_FILES)"
-	$(foreach prerequisite,$^,$(call execute-command,cp -RLf $(prerequisite) $@))
+	cp -RLf $< $@
+
+ifdef ANALYTICS_SENTRY_TOKEN
+	./scripts/build/jq-inplace.sh \
```

Comment:
Given the limited use of `jq-inplace.sh`, I wonder if it'd be clearer to name it e.g. `jq-insert.sh`, and then pass it the property to insert (`analytics.sentry.token`) and value to insert (`$(ANALYTICS_SENTRY_TOKEN)`) as separate parameters? \*shrug\*